### DISCUSSION TITLE
Ctrl+R, Shift+Ctrl+R, F5 all do not reload the page

### DISCUSSION THREAD

**Discussion by albakhtari** [cstayyab/WALC#155] [Troubleshooting] [answered]
Ctrl+R, Shift+Ctrl+R, F5 all do not reload the page, instead I have to go into the page console and type: `document.location.reload()` , It worked with the previous version of WALC, I used to press Shift+Ctrl+R and the page would reload and go back to the home page of web.whatsapp.com

**Comment by daffamumtaz2361** [comment, marked as answer]
Yeah, that was causing a bug. It basically disconnect the app from the whatsapp web, so the tray icon and other feature relying on the connection become broken.

I kinda fixed this, but the window closes when you're reloading, a few times actually. If you disabled the tray and ask on exit, the app just straight up closes. If you just disabled the tray, the exit dialog will repeatedly appear. That's why I just disabled reloading entirely.

Would you like for a restart feature instead? It will restart the app entirely instead of just reloading, would probably be slightly longer though.

**Reply by albakhtari** [reply]
That's actually exactly what I use it for I reload then close it, so it's perfect!
Thanks @Zzombiee2361

**Reply by daffamumtaz2361** [reply]
That's still not what I'd call normal behavior, so I won't be adding a reload button. Don't forget the fact that if you disabled the tray/close to tray it will annoyingly ask if you want to exit repeatedly, and straight up closes if you also disabled ask on exit.

Besides, why are you even reloading the page to begin with? If this is something you rarely do because whatsapp web is a little buggy, an app restart button could suffice.

**Reply by albakhtari** [reply]
The reason I reload the page is so that I'm not on my last opened chat when I open it again.
So the reload function for me is basically a WhatsApp web home button.
So anything that closes the open chat will do for me.

**Reply by daffamumtaz2361** [reply]
That's a very band aid solution. You know what I'll tell you about my solution to this problem. So it started with my friend adding me to a group then immediately kicked me out of it. He says he want to use that group for personal notes and checking connection. What a great idea, I thought.

So then I created group with me and him then immediately kicked him. I pinned the group and make sure it stays on top. Everytime I finished with a chat I open that group. You can also share clipboard between your PC and phone. You should try it. Just don't store your private stuff there, maybe make a 2nd group for that.

Also did you know that WALC has chat shortcut? Press Ctrl + 1-9 to open the chat. It's a little buggy at the moment, but it works well enough.

**Reply by albakhtari** [reply]
In my case, I didn't make a group with my self, instead I clicked on this link https://api.WhatsApp.com/send?phone=MyPhoneNumber and pinned the chat.
Works like a charm, but you'll see your chat name as your phone number.
You can also see what your profile tab looks like in other people's phones.
The only thing is that once you send your self a message and it goes through, the message will automatically become with two blue ticks.


https://user-images.githubusercontent.com/79008923/132836806-89a55fa1-a2bc-4757-bcab-140625be3f11.mp4

> Everytime I finished with a chat I open that group.

I would still prefer to start WALC with the WhatsApp web home screen.
